cengage/react-magma
# feat(Tabs): New `TabsScrollSpyContainer` and `TabScrollSpyPanel` components which allow for a unique `Tab` layout with a scroll-spy navigation UI.

Issue: # [1144](https://github.com/cengage/react-magma/issues/1144)

## What I did
- New `TabsScrollSpyContainer` and `TabScrollSpyPanel` components have been added for the scroll spy feature in the `Tabs` component.

## Screenshots:
https://github.com/cengage/react-magma/assets/77400920/d99c4629-eb47-4648-b1db-3a94c64042a2


## Checklist 

## How to test

- Open Storybook example of Tabs - Scroll Spy.
- Ensure the first navigation item (Card 1) is active.
- Scroll down to have "Area 2" in view.
- Ensure the second navigation item (Card 2) is active.
- Continue to verify that the Navigation items update each area as you scroll to the bottom.
- Click on any navigation item and confirm that it updates the active state and shows the associated content on the right.
- Open Storybook example of Tabs - Scroll Spy Icons.
- Verify same behavior is consistent as you scroll with the non icon version.
- Make sure isInverse works on both Storybook examples.
